ee/tabby-webserver/src/schema/auth.rs

```diff
@@ -463,6 +463,8 @@ pub trait AuthenticationService: Send + Sync {
     async fn delete_oauth_credential(&self, provider: OAuthProvider) -> Result<()>;
     async fn update_user_active(&self, id: &ID, active: bool) -> Result<()>;
     async fn update_user_role(&self, id: &ID, is_admin: bool) -> Result<()>;
+    async fn update_user_avatar(&self, id: &ID, avatar_base64: Option<String>) -> Result<()>;
```

Comment:
```suggestion
    async fn update_user_avatar(&self, id: &ID, avatar: Option<Vec<u8>>) -> Result<()>;
```

Let's use blob format in service level